Web/Scotty/Body.hs

```diff
@@ -64,7 +64,10 @@ getFormParamsAndFilesAction istate prbo req bodyInfo opts = do
   let
     wholeBody = BL.toChunks bs
   (formparams, fs) <- parseRequestBodyExBS istate prbo wholeBody (W.getRequestBodyType req) `catches` handleWaiParseSafeExceptions
-  return (convertBoth <$> formparams, convertKey <$> fs)
+  let 
+    fs' = filter emptyFile fs
+  return (convertBoth <$> formparams, convertKey <$> fs')
+    where emptyFile (_,fInfo) = ("\"\"" /= (W.fileName fInfo))
```

Comment:
How do HTML forms behave if the `name` attribute is absent? i.e. how does the payload sent by the browser look like.

Reply:
Hi @ocramz,
I will add the test-cases.

payload when I upload style.css file with `name` attribute in HTML.
![Screenshot from 2024-04-30 12-10-21](https://github.com/scotty-web/scotty/assets/40828350/d65f936f-c05d-4883-8a23-108795259529)

Payload when I upload style.css file without `name` attribute in HTML.
![image](https://github.com/scotty-web/scotty/assets/40828350/9409ac6a-2ec6-4013-8b3f-7a94d6e959ba)